plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/utils.tsx

```diff
@@ -0,0 +1,26 @@
+export const checkOverflow = ( scrollableElement: HTMLElement ) => {
+	const marginError = 3;
```

Comment:
I think the `Error` suffix didn't help me understand what's this for 😅. Is `overflowThreshold` more accurate? Also, is this in pixels? A comment for posterity would be helpful.